packages/engine/Source/Shaders/Model/ModelVS.glsl

```diff
@@ -104,7 +104,11 @@ void main()
 
     // Compute the final position in each coordinate system needed.
     // This returns the value that will be assigned to gl_Position.
-    vec4 positionClip = geometryStage(attributes, modelView, normal);    
+    vec4 positionClip = geometryStage(attributes, modelView, normal);
+
+    #ifdef HAS_FOG
```

Comment:
I would add a note that this must go after the geometry stage as it uses v_positionWC